package.json

```diff
@@ -350,7 +350,7 @@
     "webpack-cli": "3.3.12"
   },
   "dependencies": {
-    "@brave/leo": "github:brave/leo",
+    "@brave/leo": "github:brave/leo#skia-icons",
```

Comment:
**Note:** Before merging this should be updated to point at the Leo ToT but https://github.com/brave/leo/pull/222 needs to land first.